core/templates/filters/translate.pipe.spec.ts

```diff
@@ -117,6 +117,7 @@ describe('TranslatePipe', () => {
 // I18n_rogue_1: '<script>alert(\'Oppia\');</script>Hello',
 // I18n_rogue_2: '<oppia-img>Me</oppia-img>Hola'.
 @Component({
+  /* eslint-disable oppia/no-multiline-disable */
```

Comment:
@Hudda what is this? We should not be disabling our disable checks, that kind of defeats the whole point. I don't think you even need to break the max-len convention at all here, it is not required that each line of the template be on a single line right?